src/Arcus.Messaging.ServiceBus.Core/Extensions/MessageExtensions.cs

```diff
@@ -50,7 +50,7 @@ public static MessageCorrelationInfo GetCorrelationInfo(this Message message)
 
         private static string DetermineOperationId(string messageCorrelationId)
         {
-            if (String.IsNullOrWhiteSpace(messageCorrelationId))
+            if (string.IsNullOrWhiteSpace(messageCorrelationId))
```

Comment:
😄 I use the exact opposite all the time. To make a diff between the type and the variable.